lib/coreos.py

```diff
@@ -66,11 +66,22 @@ def read_conf(cluster_name, path):
     )
 
     for v in c['volumes']:
-        conf = conf.volume(**v)
+        conf = conf.volume(**camelize_dict(v))
```

Comment:
Thinking it might be better to move camilization to the `cluster_conf`. I think snake case is more pythonic and we need camel case just before we send things to boto.